src/super_gradients/training/models/segmentation_models/segformer.py

```diff
@@ -372,12 +386,58 @@ def _remove_auxiliary_heads(self):
     def replace_head(self, new_num_classes: int, new_decoder_embed_dim: int):
         self.decode_head = SegFormerHead(encoder_dims=self.encoder_embed_dims, embed_dim=new_decoder_embed_dim, num_classes=new_num_classes)
 
-    def forward(self, x: torch.Tensor) -> torch.Tensor:
+    def _forward(self, x: torch.Tensor) -> torch.Tensor:
         features = self._backbone(x)
         out = self.decode_head(features)
         out = F.interpolate(out, size=x.shape[2:], mode="bilinear", align_corners=False)
         return out
 
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        if self.is_SWI:
+            return self.sliding_window(x, self.num_classes, self.sliding_window_stride, self.sliding_window_size)
```

Comment:
Seems like crop_size and stride args are redundant if we always use the attributes.

Reply:
Fixed on c79d0f2